packages/familie-backend/src/auth/bruker.ts

```diff
@@ -16,3 +17,38 @@ export const hentBrukerprofil = () => {
         res.status(200).send(user);
     };
 };
+
+// Hent brukerenhet
+export const hentBrukerenhet = (scope?: string) => {
```

Comment:
We can add enhet to the session and just return it in the `hentBrukerprofil` function. So when the session is created we can fetch enhet once.

Reply:
ok